src/components/GanttChart.vue

```diff
@@ -143,6 +154,20 @@ export default {
     },
   },
 
+  watch: {
+    period: {
+      async handler() {
+        this.$nextTick(() =>
+          this.$refs.period.scrollIntoView({ inline: 'center', behavior: 'smooth' })
```

Comment:
Bør sjekke at `this.$refs.period` eksisterer; spytter ellers ut litt errors ved valg av "Alt".